Dockerfile

```
@@ -184,13 +183,11 @@ RUN curl -fsSL https://github.com/lh3/minimap2/releases/download/v2.24/minimap2-
 ARG CACHE_DATE
 
 # Download Nextclade v3
-# Note: Before v3 there used to be separate binaries for Nextclade and
-# Nextalign. Since v3 there is only Nextclade, which can do both.
-# TODO: After Nextclade 3 is released, update the URL to download the latest
-# version instead of hardcoded.
-# TODO: At some point, update the v2 binary symlinks to use Nextclade v3.
-RUN curl -fsSL https://github.com/nextstrain/nextclade/releases/download/3.0.0-alpha.2/nextclade-$(/builder-scripts/target-triple) \
-  -o /final/bin/nextclade3
+# Set default Nextclade version to 3
+RUN VERSION=$(curl -fsSL https://api.github.com/repos/nextstrain/nextclade/releases/latest | \
+  jq --raw-output '.tag_name') && curl -fsSL https://github.com/nextstrain/nextclade/releases/download/"$VERSION"/nextclade-$(/builder-scripts/target-triple) \
```

```suggestion
  jq --raw-output '.tag_name') && curl -fsSL https://github.com/nextstrain/nextclade/releases/latest/download/nextclade-$(/builder-scripts/target-triple) \
```
No need for jq this way

---

Oh yeah, I vaguely remember this... `/releases/download/<tag>` vs. `/releases/latest/download`, confusing 😵‍💫

Trying in 3ee286c182564b2bb7b282ff6d27666d53656f02

---

I guess it makes sense since a tag named `latest` is valid

---

Yeah I got confused as well - good point that this is possibly to work around `latest` being a valid tag, didn't think of this